lapis_lazuli.gemspec

```diff
@@ -44,8 +44,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "facets", "~> 3.1" # Used in the cucumber project generator
 
   # webdriver specifics
-  spec.add_dependency "selenium-webdriver", "~> 3"
+  spec.add_dependency "selenium-webdriver", "> 2"
```

Comment:
Why not force a newer selenium version?

Reply:
Because we support both version 2 and version 3. If someone wants to use the old version (for testing on an old browser for example), we don't want to force them. Next patch will include `'~>2', '~>3'